### ISSUE TITLE
URL config files

### ISSUE THREAD

**Issue by nsheff** [pepkit/peppy#362]
I'd like to be able to do this:

```
import peppy
peppy.Project("https://raw.githubusercontent.com/pepkit/example_peps/master/example_basic/project_config.yaml")
```

what do you think?

**Comment by nsheff** [comment]
Also, what about this?

```
pep_version: "2.0.0"
import:
  https://raw.githubusercontent.com/pepkit/example_peps/master/example_basic/project_config.yaml
  ```

**Comment by stolarczyk** [comment]
done 

```python
In [1]: import peppy                                                                                                                                                

In [2]: p=peppy.Project("https://raw.githubusercontent.com/pepkit/example_peps/master/example_basic/project_config.yaml")                                           

In [3]: print(p)                                                                                                                                                    
Project 'example_basic' (https://raw.githubusercontent.com/pepkit/example_peps/master/example_basic/project_config.yaml)
2 samples: frog_1, frog_2
Sections: pep_version, sample_table
```

**Comment by khoroshevskyi** [comment]
This issue was solved [0f3c371](https://github.com/pepkit/peppy/commit/0f3c371df65b7ae9d16cf4ed7a29f2563098c6a7)
